maze_transformer/generation/constants.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from muutils.tensor_utils import NDArray
+
+Coord = NDArray["x y", np.int8]
+CoordTup = tuple[int, int]
+CoordArray = NDArray["coords", np.int8]
+
+SPECIAL_TOKENS: dict[str, str] = dict(
+    adjlist_start="<ADJLIST_START>",
+    adjlist_end="<ADJLIST_END>",
+    target_start="<TARGET_START>",
+    target_end="<TARGET_END>",
+    origin_start="<ORIGIN_START>",
+    origin_end="<ORIGIN_END>",
+    path_start="<PATH_START>",
+    path_end="<PATH_END>",
+    connector="<-->",
+    adjacency_endline=";",
+    padding="<PADDING>",
+)
```

Comment:
I'm not very familiar with Python enums - what would be the advantage over a dict?

(Also yeah probably out of scope)

Reply:
Agree that this would be a good idea but also agree that it might be out of scope. Enums would mean that:
- type checkers/linters would let us know that we are doing so
- you'd get suggestions in your IDE when trying to get `SPECIAL_TOKENS.adjlist_start`, instead of having to index into the dict and remember the correct name